UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Filters/AzureDependencyFilterTelemetryProcessor.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.ExchangeSetService.FulfilmentService.Filters
+{
+    [ExcludeFromCodeCoverage]
```

Comment:
This is perfectly testable so no need to exclude from code coverage and please add some quick unit tests.

Reply:
Thanks for highlighting, added unit tests cases.